.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,42 @@
+# This workflow file builds and pushes the Dockerfile
+# located in this repo, to the SCD harbor Docker registry.
+# It uses a harbor robot account, who's credentials are 
+# stored as secrets in this repo.
+
+name: Build & Push Docker Image
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - feature/adding-docker-file
```

Comment:
Once this PR is merged into `feature/k8s-deployment`, this workflow will no longer trigger. If we want it to trigger then we need to change this.